MIGRATION_GUIDE.md

```diff
@@ -133,3 +136,43 @@ const Home = ({ navigation }: { navigation: any }) => {
  ...
 };
 ```
+
+### Plugins
+
+**The plugins for V2 have changed from V1**. 
+
+The plugins have been re-released with different names. These are the equivalent packages for V2. Not all packages in V1 have yet been released for V2 but Segment is actively adding more packages to the list. 
+
+Also review [the main package list](README.md#supported-plugins) for new V2 plugins.
+
+| Plugin | V1 Package      | V2 Package     |
+| ----------- | ----------- | ----------- |
+| [Adjust](https://github.com/segmentio/analytics-react-native/tree/master/packages/plugins/plugin-adjust)      | `@segment/analytics-react-native-adjust`|  `@segment/analytics-react-native-plugin-adjust` |
+| [Amplitude Sessions](https://github.com/segmentio/analytics-react-native/tree/master/packages/plugins/plugin-amplitudeSession)      | `@segment/analytics-react-native-amplitude`| `@segment/analytics-react-native-plugin-amplitude-session`|
+| [AppsFlyer](https://github.com/segmentio/analytics-react-native/tree/master/packages/plugins/plugin-appsflyer)    | `@segment/analytics-react-native-appsflyer` | `@segment/analytics-react-native-plugin-appsflyer`|
+| [Facebook App Events](https://github.com/segmentio/analytics-react-native/tree/master/packages/plugins/plugin-facebook-app-events)    | `@segment/analytics-react-native-facebook-app-events-ios` | `@segment/analytics-react-native-plugin-facebook-app-events` |
+| [Firebase](https://github.com/segmentio/analytics-react-native/tree/master/packages/plugins/plugin-firebase)      | `@segment/analytics-react-native-firebase` | `@segment/analytics-react-native-plugin-firebase`|
+| [Mixpanel](https://github.com/segmentio/analytics-react-native/tree/master/packages/plugins/plugin-mixpanel)    | `@segment/analytics-react-native-mixpanel` | `@segment/analytics-react-native-plugin-mixpanel` |
+| [Taplytics](https://github.com/taplytics/segment-react-native-plugin-taplytics)     | `@segment/analytics-react-native-taplytics-ios` | `@taplytics/segment-react-native-plugin-taplytics` |
+
+### Context Traits
+
+Previous versions of this library used to persist `traits` accross events after an `identify` was sent inside `context.traits` we're not supporting this out of the box in V2 and the rest of our mobile libraries due concerns of data privacy for some device destinations. Some device mode destinations might not accept events if they contain identifiable data.
```

Comment:
Previous versions of this library used to persist `traits` in `context.traits` across all event types after an `identify` event was sent.  We no longer not support this out of the box in V2 as well as the rest of our mobile libraries due to concerns of data privacy for some device mode destinations. Some device mode destinations might not accept events if they contain identifiable data as it is against their Terms of Service.